forte/pipeline.py

```diff
@@ -308,30 +312,53 @@ class RequestBody(BaseModel):
         # pylint: disable=unused-variable
         @app.get("/")
         def default_page():
-            return {"status": "OK", "pipeline": self._dump_to_config()}
+            nonlocal name
+            return {
+                "status": "OK",
+                "name": name,
+                "pipeline": self._dump_to_config(),
+            }
+
+        @app.get("/records")
+        def get_records():
+            nonlocal records
+            if not records:
+                # Collect records of each pipeline component for validation
+                meta_records: Dict = {}
+                for component in [self._reader] + self.components:
+                    component.record(meta_records)
+                records = json.dumps(
+                    {k: list(v) for k, v in meta_records.items()}
+                )
+            return {"status": "OK", "records": records}
 
         @app.post("/process")
         def run_pipeline(body: RequestBody):
             args = json.loads(body.args)
             kwargs = json.loads(body.kwargs)
             result = self.process(*args, **kwargs)
-            return {"result": result.serialize()}
+            return {"status": "OK", "result": result.serialize()}
 
         # pylint: enable=unused-variable
 
         return app
 
-    def serve(self, host: str = "localhost", port: int = 8008):
+    def serve(self, host: str = "localhost", port: int = 8008, name: str = ""):
         r"""Start a service of the current pipeline at a specified host
         and port.
 
         Args:
             host: Port number of pipeline service.
             port: Host name of pipeline service.
+            name: Assign a name to the pipeline service for validation.
+                Default to `''`.
         """
         self.initialize()
         uvicorn.run(
-            self._remote_service_app, host=host, port=port, log_level="info"
+            self._remote_service_app(name=name),
+            host=host,
+            port=port,
+            log_level="info",
```

Comment:
I observed `uvicorn` duplicate every error messages, maybe the logging setup is not correct?

```
INFO:     Started server process [14115]
INFO:uvicorn.error:Started server process [14115]
INFO:     Waiting for application startup.
INFO:uvicorn.error:Waiting for application startup.
INFO:     Application startup complete.
INFO:uvicorn.error:Application startup complete.
INFO:     Uvicorn running on http://localhost:8008 (Press CTRL+C to quit)
INFO:uvicorn.error:Uvicorn running on http://localhost:8008 (Press CTRL+C to quit)
```